samples/msal-node-samples/client-credentials/package.json

```diff
@@ -10,6 +10,6 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "@azure/msal-node": "file:../../../../lib/msal-node"
+    "@azure/msal-node": "file: ../../lib/msal-node"
```

Comment:
Should we default to the published version here, like the other samples?

Reply:
Thanks for catching this, I missed the ones that don't have tests on them yet.